docs/block-api.md

```diff
@@ -93,5 +93,5 @@ className: false,
 
 ## Edit and Save
 
-The `edit` and `save` functions define the editor interface with which a user would interact, and the markup to be serialized back when a post is saved. They are the heart of how a block operates, so we'll cover them separately.
+The `edit` and `save` functions define the editor interface with which a user would interact, and the markup to be serialized back when a post is saved. They are the heart of how a block operates, so they are [covered separately](/block-edit-save/).
```

Comment:
Might consider a relative link in case docs are hosted in a sub folder, this will break.

Reply:
It didn't work properly when I tried it. (There's a link in one of the other docs that looked broken to.)